index.bs

```diff
@@ -816,62 +868,111 @@ To <dfn>handle funky elements</dfn> on a given |element|, run these steps:
-<dfn>effective attribute configuration</dfn> for an attribute |attr|
-attached to an element |element|, run these steps:
-  1. Let |config| be |sanitizer|'s [=configuration object=].
-  1. Let |baseline action| be the result of running the steps of the
-     [=determine the baseline configuration for an attribute=] algorithm
-     on the attribute |attr|.
-  1. Let |config action| be the result of running the steps of the
-     [=determine the effective configuration for an attribute=] algorithm
-     on the attribute |attr|, with the element |element| and the
-     config |config|.
-  1. Return the [=stricter action=] of |baseline action| and |config action|.
+<div algorithm="attribute matches an attribute match list">
+To determine whether an <dfn>|attribute| matches an [=attribute match
+list=]</dfn> |list|, run these steps:
+
+  1. If |attribute|'s [=Attr/local name=] does not match the
+     [=attribute match list=] |list|'s
+     [key](https://webidl.spec.whatwg.org/#idl-record) and if the key is
+     not `"*"`: Return `false`.
+  1. Let |element| be the |attribute|'s {{ownerElement}}.
+  1. Let |element name| be |element|'s [=Element/local name=].
+  1. If |element| is in either the [=SVG namespace|SVG=] or
+     [=MathML namespace|MathML=] namespaces, prefix |element name| with
+     the appropriate [[#namespaces|namespace designator]] plus a whitespace
+     character.
+  1. If |list|'s [value](https://webidl.spec.whatwg.org/#idl-record) does not
+     contain |element name| and value is not `["*"]`: Return `false`.
+  1. Return `true`.
+
+
+Note: The element names in the Sanitizer configuration are normalized according
+  to normalization step in the HTML Parser, just like elements'
+  [=Element/local names=] are. Thus, the comparison is effectively case
+  insensitive.
 </div>
 
-Before describing how an effective configuration is derived, we need a
-helper definition:
+<div algorithm="sanitize action for an attribute">
+To determine the <dfn>sanitize action for an |attribute|</dfn> given a Sanitizer
+configuration dictionary |config|, run these steps:
+
+  1. Let |kind| be |attribute|'s [=attribute kind=].
+  1. If |attribute| is of `regular` |kind| and its name does not match any
+     name in the [=baseline attribute allow list=]: Return `drop`.
+  1. If |attribute| [=attribute matches an attribute match list|matches=] any
+     [=attribute match list=] in |config|'s [=attribute drop list=]: Return
+     `drop`.
+  1. If |config| has a non-empty [=attribute allow list=] and if |attribute|
+     does not [=attribute matches an attribute match list|match=] any
+     [=attribute match list=] in |config|'s [=attribute allow list=]: Return
+     `drop`.
+  1. If |config| has an empty [=attribute allow list=] and if |attribute|
+     does not [=attribute matches an attribute match list|match=] any
+     [=attribute match list=] in the [=default configuration=]'s
+     [=attribute allow list=]: Return `drop`.
```

Comment:
Same suggestion here as with the element allow list above about avoiding the ifs by making the attribute allow list have the right contents.

Reply:
Done.